Cargo.toml

```diff
@@ -41,7 +41,7 @@ chrono = { version = "0.4.26", features = ["serde"] }
 uuid = {version = "1.3.4", features = ["v4", "serde"]}
 reqwest = { version = "0.11.18", features = ["json"]}
 jsonschema = "~0.17"
-jsonlogic = "0.5.1"
+jsonlogic = { git = "https://github.com/juspay/jsonlogic_rs.git", rev = "4f1a8bcbb423c599eacf6c95ae17f5c0974bc071" }
```

Comment:
Why did we make this change? All other changes are in JS

Reply:
to add the same operators in the rust client

Reply:
Are those added in this PR? If not, can we make this change when we add the operations

Reply:
https://github.com/juspay/jsonlogic_rs/pull/1